{
    "title": "lighttpd 1.4.18, and possibly other versions before 1.5.0, does not properly calculate the size of a file descriptor array, which allows remote attackers to cause a denial of service (crash) via a large number of connections, which triggers an out-of-bounds access.",
    "score": 5.0,
    "severity": "MEDIUM",
    "cwe_id": "CWE-399",
    "cvss_vector": "CVSS:2.0/AV:N/AC:L/Au:N/C:N/I:N/A:P",
    "cpe_match": [
        "cpe:2.3:a:lighttpd:lighttpd:1.4.18:*:*:*:*:*:*:*",
        "cpe:2.3:a:lighttpd:lighttpd:1.4.8:*:*:*:*:*:*:*",
        "cpe:2.3:a:lighttpd:lighttpd:1.4.17:*:*:*:*:*:*:*",
        "cpe:2.3:a:lighttpd:lighttpd:1.4.11:*:*:*:*:*:*:*",
        "cpe:2.3:a:lighttpd:lighttpd:1.4.14:*:*:*:*:*:*:*",
        "cpe:2.3:a:lighttpd:lighttpd:1.4.10:*:*:*:*:*:*:*",
        "cpe:2.3:a:lighttpd:lighttpd:1.4.16:*:*:*:*:*:*:*",
        "cpe:2.3:a:lighttpd:lighttpd:1.4.12:*:*:*:*:*:*:*",
        "cpe:2.3:a:lighttpd:lighttpd:1.4.9:*:*:*:*:*:*:*",
        "cpe:2.3:a:lighttpd:lighttpd:1.4.7:*:*:*:*:*:*:*",
        "cpe:2.3:a:lighttpd:lighttpd:1.4.15:*:*:*:*:*:*:*",
        "cpe:2.3:a:lighttpd:lighttpd:1.4.13:*:*:*:*:*:*:*"
    ],
    "cve_source": [
        "NVD"
    ],
    "ttpMapping": []
}
